components/eamxx/tests/uncoupled/surface_coupling/surface_coupling.cpp

```diff
@@ -260,20 +262,25 @@ void test_imports(const FieldManager& fm,
     EKAT_REQUIRE(surf_evap(i)       == import_constant_multiple_view(13)*import_data_view(i, import_cpl_indices_view(13)));
 
     // The following are only imported during run phase. If this test is called
-    // during initialization, all values should still be 0.
+    // during initialization, all values should be the default value.
+    // TODO: Why are some of these FillValue and others are 0.0?  For the former,
```

Comment:
@tcclevenger , do you have any thoughts on the behavior I am describing in this TODO comment?

Reply:
This is certainly a bug. Maybe it's related to the `curr_fill & new_fill` if statement above?

Reply:
so weird, I swear I changed that in the latest commit.... oh well, git mysteries.  I'll fix it again and see if that also fixes the SC test.